packages/flutter_app/ios/prod/ExportOptions.plist

```diff
@@ -4,17 +4,14 @@
 	<dict>
 		<key>destination</key>
 		<string>export</string>
+		<key>generateAppStoreInformation</key>
+		<false/>
+		<key>manageAppVersionAndBuildNumber</key>
+		<false/>
```

Comment:
This is optional, but I added it incidentally.
I prefer that the build number is not incremented by itself, so I set it to false.